packages/wrangler/src/__tests__/helpers/mock-get-pages-upload-token.ts

```diff
@@ -0,0 +1,22 @@
+import { rest } from "msw";
+import { msw } from "./msw";
+
+export function mockGetUploadToken(
```

Comment:
Mind adding a docstring for what this does?

Reply:
✅